sample_consensus/include/pcl/sample_consensus/impl/sac_model_registration.hpp

```diff
@@ -55,9 +55,12 @@ pcl::SampleConsensusModelRegistration<PointT>::isSampleGood (const Indices &samp
   using namespace pcl::common;
   using namespace pcl::traits;
 
-  PointT p10 = (*input_)[samples[1]] - (*input_)[samples[0]];
-  PointT p20 = (*input_)[samples[2]] - (*input_)[samples[0]];
-  PointT p21 = (*input_)[samples[2]] - (*input_)[samples[1]];
+  PointT p10;
+  pcl::copyPoint((*input_)[samples[1]] - (*input_)[samples[0]], p10);
+  PointT p20;
+  pcl::copyPoint((*input_)[samples[2]] - (*input_)[samples[0]], p20);
+  PointT p21;
+  pcl::copyPoint((*input_)[samples[2]] - (*input_)[samples[1]], p21);
```

Comment:
Unrelated?

Reply:
I believe this is to fix an error while compiling with PointXYZLAB, I noticed that too while compiling yesterday. But I think a better fix would be to use the Eigen mappings `Eigen::Vector3f p10 = (*input_)[samples[1]].getVector3fMap() - (*input_)[samples[0]].getVector3fMap();` and/or to add an additional [constructor like the other point types have](https://github.com/PointCloudLibrary/pcl/blob/master/common/include/pcl/impl/point_types.hpp#L749). Also we should do that in a separate pull request to get that merged as soon as possible.

Reply:
>  Also we should do that in a separate pull request to get that merged as soon as possible.

Seconded

Reply:
reverted